cmake/Modules/UseOpenSSL.cmake

```diff
@@ -472,9 +472,10 @@ function(connextdds_openssl_generate_signed_certificate)
         _OPENSSL_CA_KEY_FILE _OPENSSL_CA_CONFIG_FILE _OPENSSL_CA_CERT_FILE
     )
 
-    # The cert request file is just the cert file with "Cert" replaced by "Req".
+    # The cert request file is just the cert file with "_cert" replaced by
+    # "Req".
     set(certRequestFile)
-    string(REPLACE "Cert" "Req" certRequestFile "${_OPENSSL_OUTPUT_CERT_FILE}")
+    string(REPLACE "_cert" "Req" certRequestFile "${_OPENSSL_OUTPUT_CERT_FILE}")
```

Comment:
Sorry about this change. The certificates in the community examples in develop have `_cert` in their names.
I had updated the certificate names to use `Cert` instead, because that's what we are currently using for the artifacts that we ship in Connext.
However, these changes in the name are part of [a PR in the Community examples](https://github.com/rticommunity/rticonnextdds-examples/pull/524/) that is not yet in develop. I'll create an issue to update this line to "Cert" just before merging the examples PR.

Reply:
Following @lulivi's recommendation, I made sure that the affected examples [work after the changes](https://github.com/rticommunity/rticonnextdds-examples/search?q=include%28ConnextDdsGenerateSecurityArtifacts%29) in this PR.

Having git pointing to develop in the GitHub examples repository, but pointing to feature/67/update-UseOpenSSL in the rticonnextdds-cmake-utils submodule, I can compile and run successfully the network capture security examples, and the  real_time_wan_transport example. The latter I was able to create the participants, but didn't get to a successful communication because I haven't setup a static NAT binding. However, UseOpenssl.cmake should affect that.